templates/rhpam74-authoring-ha.yaml

```diff
@@ -255,7 +255,7 @@ parameters:
   displayName: Image
   name: AMQ_BROKER_IMAGE
   required: true
-  value: registry.redhat.io/amq-broker-7/amq-broker-72-openshift:1.3
+  value: registry.redhat.io/amq-broker-7-tech-preview/amq-broker-72-scaledown-openshift:0.4
```

Comment:
We cannot replace the broker with the scaledown controller here.

Reply:
@errantepiphany sorry! my bad. I will fix that.

Reply:
@msmagnanijr did you forget to update the scaledown image?
https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/templates/rhpam74-authoring-ha.yaml#L244-L253